Fix typos in RUNPATH_FILE description (#8397)
equinor/ert: docs/reference/configuration/keywords.rst

------------
.. _runpath_file:

When running workflows based on external scripts, it is necessary to 'tell' the
external script where all the realisations are located in
the filesystem. Since the number of realisations can be quite high this will

> easily overflow the commandline buffer; the solution used is

Reviewer: ```suggestion
easily overflow the commandline buffer; the solution is
to let ERT write a file which looks like this::
```

> easily overflow the commandline buffer; the solution used is
> to let ERT write a regular file. It looks like this::

Reviewer: ```suggestion
easily overflow the commandline buffer; the solution is
to let ERT write a file which looks like this::
```

Reviewer: Jus a small suggestion :)

Author: Thanks Eivind. I would keep as it is just because I am too lazy to make a rebase on web version of Github :-)
